Makefile

```diff
@@ -102,7 +103,8 @@ COVERAGEFLAGS = $(OBJ_DIR_TEST) --object-directory=$(shell pwd) --root=${ROOT} \
                 --exclude='(^modules/gmock/|^modules/cpp-common/include/|^modules/cpp-common/test_utils/|^ut/)' \
                 --sort-percentage
 
-EXTRA_CLEANS += $(TEST_XML) \
+EXTRA_CLEANS += *.o *.so *.d \
```

Comment:
No, since we call `rm` without `-f` (for sanity :grinning:).

Also, there are no `init.d` files in this repo.